### ISSUE TITLE
Opbuntu

### ISSUE THREAD

**Issue by tri-pole** [ophub/amlogic-s9xxx-armbian#573]
@ophub 

I know this place is for technical discussion and if you think that my post is inappropriate or out-of-place I apologize and I fully understand if you remove it.

By now, Ubuntu for ARM64 can be made to run very well on Amlogic TV-boxes with ophub/flippy/unifreq kernels (and even linux-image-generic).

Therefore, it may be worth considering using Ubuntu as base for your builds for TV-boxes instead of Armbian, and probably you already have considered it.

I think it could provide several benefits:

It is very easy to build a basic system from the daily builds of the Ubuntu rootfs, see [template/example here](https://gitlab.com/tripole-inc/bootdisk/-/blob/main/ubuntu_mkimg.bash) and by basing the images directly on Ubuntu you could make "Obpuntu" (or whatever you call your flavor) an independent "brand" of its own. (Then various small parts from other distros, like Armbian, could be added if considered to be useful.)

**Comment by ophub** [comment]
This is a great proposal, and I see that the project you posted has done a good exploratory practice.
I am a layman with insufficient professional knowledge. I need to continue to study. After mastering the relevant knowledge, I will continue to improve.

**Comment by tri-pole** [comment]
Thank you for your response.

I too am a layman and my knowledge is very limited indeed; I often look at others (like your project).

I have been very impressed with the development of amlogic-s9xxx-armbian and the community that is built around it. Linux users with Amlogic boxes have found a new home here, I think. This is very good because the power of the TV-boxes is
a bit under-utilized when running Android, in my opinion.

I will continue to explore, in small steps, the idea of using Ubuntu as a base but I will never make anything big from it, like developing a distro. It is my hope that other will catch on and develop the idea further.

I think you can close this thread now, if you wish. Again, thanks for your great contributions.

**Comment by ophub** [comment]
Dreams come true. I opened this warehouse on September 19 last year, and it's been exactly one year now.

I have learned and practiced little by little, and there are still a lot of them who can’t. But looking back at a year ago, I have made great progress. As long as you keep exploring, you will definitely be able to. Being able to come up with innovative ideas is the greatest foundation for progress, and a way to achieve it will always be found.
